command.h

```diff
@@ -106,6 +106,10 @@ enum event_command
    CMD_EVENT_HISTORY_DEINIT,
    /* Initializes history playlist. */
    CMD_EVENT_HISTORY_INIT,
+   /* Deinitializes favourties playlist. */
+   CMD_EVENT_FAVORITES_DEINIT,
+   /* Initializes favourties playlist. */
+   CMD_EVENT_FAVORITES_INIT,
```

Comment:
Perhaps best not to add user-oriented actions like this to command event. Instead, just add functions to retroarch.c, and add function prototypes to retroarch.h.

Reply:
Just pushed a commit that fixes this  :)